development/gcp/pkg/pubsub/types.go

```diff
@@ -66,3 +66,19 @@ type FailingTestMessage struct {
 	GithubCommitersLogins []string `json:"githubCommitersLogins,omitempty"`
 	CommitersSlackLogins  []string `json:"slackCommitersLogins,omitempty"`
 }
+
+type Rotation struct {
+	nextRotationTime string `yaml:"nextRotationTime"`
+	rotationPeriod   string `yaml:"rotationPeriod"`
+}
```

Comment:
Will this really work? From what I remember you can't unmarshal value into a variable when it's not exported. You have to export those values to get information you need.